resources/views/manager/index.blade.php

```diff
@@ -32,14 +32,15 @@
                 <div class="container mx-auto px-4">
                     <div class="md:flex">
                         <div class="flex -mb-px mr-8">
-                            <router-link to="/" class="no-underline text-white  opacity-50 md:text-grey-dark md:opacity-100 flex items-center py-4 border-b hover:opacity-100 md:hover:border-gob-gold">
+                            <router-link to="/dashboard" class="no-underline text-white  opacity-50 md:text-grey-dark md:opacity-100 flex items-center py-4 border-b hover:opacity-100 md:hover:border-gob-gold">
```

Comment:
Remove dashboard, breaks the design route path.

Reply:
listo